src/index.ts

```diff
@@ -141,6 +143,21 @@ export function format<A extends object>(formatter: Formatter<A>, a: A, encode:
   return formatter.run(Route.empty, a).toString(encode)
 }
 
+/**
+ * @since 0.6.0
+ */
+export const getParserSemigroup = <A extends object>(): Semigroup<Parser<A>> => ({
```

Comment:
I'd remove `getParserSemigroup` in favour of `getParserMonoid` only (because there's no different constraint)

Reply:
Sure